Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    evervault (0.1.1)
+    evervault (0.1.2)
```

Comment:
Did you change this manually? If you want to bump the version we need to change the version file as well